README.md

```diff
@@ -129,28 +129,28 @@ auto message = fine::decode<std::string>(env, term);
 
 Fine provides implementations for the following types:
 
-| Type                                 | Encoder | Decoder |
-| ------------------------------------ | ------- | ------- |
-| `fine::Term`                         | x       | x       |
-| `int64_t`                            | x       | x       |
-| `uint64_t`                           | x       | x       |
-| `double`                             | x       | x       |
-| `bool`                               | x       | x       |
-| `ErlNifPid`                          | x       | x       |
-| `ErlNifBinary`                       | x       | x       |
-| `std::string_view`                   | x       | x       |
-| `std::string`                        | x       | x       |
-| `fine::Atom`                         | x       | x       |
-| `std::nullopt_t`                     | x       |         |
-| `std::optional<T>`                   | x       | x       |
-| `std::variant<Args...>`              | x       | x       |
-| `std::tuple<Args...>`                | x       | x       |
-| `std::vector<T>`                     | x       | x       |
-| `std::map<K, V>`                     | x       | x       |
-| `fine::ResourcePtr<T>`               | x       | x       |
-| `T` with [struct metadata](#structs) | x       | x       |
-| `fine::Ok<Args...>`                  | x       |         |
-| `fine::Error<Args...>`               | x       |         |
+| C++ Type                             | Elixir Type                 | Encoder | Decoder |
+| ------------------------------------ | --------------------------- | ------- | ------- |
+| `fine::Term`                         | `term`                      | x       | x       |
+| `int64_t`                            | `integer`                   | x       | x       |
+| `uint64_t`                           | `non_neg_integer`           | x       | x       |
+| `double`                             | `float`                     | x       | x       |
+| `bool`                               | `boolean`                   | x       | x       |
+| `ErlNifPid`                          | `pid`                       | x       | x       |
+| `ErlNifBinary`                       | `binary`                    | x       | x       |
+| `std::string_view`                   | `String.t()`                | x       | x       |
+| `std::string`                        | `String.t()`                | x       | x       |
```

Comment:
`std::string` can be used with any sequence of bytes (while `String.t()` is suppose to be valid utf8):

```suggestion
| `std::string_view`                   | `binary`                | x       | x       |
| `std::string`                        | `binary`                | x       | x       |
```